evm/chains/ropsten/__init__.py

```diff
@@ -1,17 +1,35 @@
 from eth_utils import decode_hex
 
+from .constants import (
+    BYZANTIUM_ROPSTEN_BLOCK,
+    SPURIOUS_DRAGON_ROPSTEN_BLOCK,
+)
 from evm import constants
+
 from evm.chains.chain import Chain
-from evm.chains.mainnet import MAINNET_VM_CONFIGURATION
 from evm.rlp.headers import BlockHeader
+from evm.vm.forks import (
+    ByzantiumVM,
+    FrontierVM,
+    SpuriousDragonVM,
+)
+
+
+ROPSTEN_VM_CONFIGURATION = (
+    # Note: Homestead and Tangerine Whistle are excluded here since VMs disallow duplicate block
+    # numbers and they are all zero for Ropsten.
+    (0, FrontierVM),
```

Comment:
This should actually be the `TangerineWhistleVM` since the ropsten network *starts* at that VM, effectively skipping over Frontier and Homestead.